src/plugins/analyze_RetraceServer.xml.in

```diff
@@ -21,7 +21,7 @@
            <_label>Insecure</_label>
            <allow-empty>yes</allow-empty>
            <_description>Whether or not to use insecure connection</_description>
-           <_note-html>Write "insecure" to allow insecure connection &lt;a href="https://fedorahosted.org/abrt/wiki/AbrtRetraceServerInsecureConnection" &gt;(warning)&lt;/a&gt;</_note-html>
+           <_note-html>Write "insecure" to allow insecure connection without SSL encryption</_note-html>
```

Comment:
I think this rather means that self-signed certificates are ignored.

Reply:
You are correct. I assumed "insecure" means no SSL, well I was wrong. I added a small patch to clarify the option on the client side as well.